websites/A/AnimesWatch/presence.ts

```diff
@@ -0,0 +1,80 @@
+const presence = new Presence({
+		clientId: "972647321970016316",
+	}),
+	getStrings = async () =>
+		presence.getStrings(
+			{
+				playing: "general.playing",
+				paused: "general.paused",
+			},
+			await presence.getSetting<string>("lang").catch(() => "tr")
+		),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let video: HTMLVideoElement, strings: Awaited<ReturnType<typeof getStrings>>;
+
+presence.on("iFrameData", async (msg: HTMLVideoElement) => {
+	if (!msg) return;
+	video = msg;
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "animeswatch",
+			details: "Ana Sayfada Göz Gezdiriyor",
+		},
+		{ pathname } = document.location,
+		[button, timestamp] = await Promise.all([
+			presence.getSetting<boolean>("button"),
+			presence.getSetting<boolean>("timestamp"),
+		]),
+		title = document.title.slice(0, document.title.length - 14);
+
+	switch (pathname.split("/")[1]) {
+		case "":
+			presenceData.details = "Ana Sayfada Göz Gezdiriyor";
+			break;
+		case "anime":
+			presenceData.details = "Bir animeye bakıyor";
+			presenceData.state = title;
+			break;
+		case "watch":
+			presenceData.details = "Bir anime izleme sayfasında";
+			presenceData.state = title;
+
+			if (video) {
+				presenceData.smallImageKey = video.paused ? "pause" : "play";
+				presenceData.smallImageText = video.paused
+					? (await strings).paused
```

Comment:
If you use the `Awaited` type as you have, there is no need to also `await` here and on other language strings
```suggestion
					? strings.paused
```